src/pages/Admin/Venue/Rooms/RoomsForm.tsx

```diff
@@ -331,6 +331,19 @@ const RoomInnerForm: React.FC<RoomInnerFormProps> = (props) => {
                       <span className="slider round"></span>
                     </label>
                   </div>
+                  <div className="toggle-room">
+                    <div className="input-title">Labeled ?</div>
+                    <label className="switch">
+                      <input
+                        disabled={disable}
+                        type="checkbox"
+                        id="isLabeled"
+                        name={"isLabeled"}
+                        ref={register}
+                      />
+                      <span className="slider round"></span>
+                    </label>
+                  </div>
```

Comment:
Same comments as above RE: using `src/components/atoms/Toggler/Toggler.tsx` here.

Reply:
Looked at this, the current implementation of `Toggler` doesn't support all the props we need it to be able to do to use in this kind of situation, so calling this OOS